### ISSUE TITLE
pytorch training problem

### ISSUE THREAD

**Issue by YZW-explorer** [Xilinx/brevitas#275]
I want to train my own model on pytorch, so I use Quantlayers to deploy my model, but when I print the parameters, I find that the model parameters are not integers
![image](https://user-images.githubusercontent.com/74363764/111617525-2ff65000-881e-11eb-9e82-efc4e31a0de1.png)

**Comment by YZW-explorer** [comment]
In fact, the essence of my question is, after I use the model built by QuantLayers for training, how do I save my model for later inference.
But Introdection wrote:
"For those users that instead are interested in deploying their quantized models, the idea obviously would be to actually gain some kind of advantage from quantization. In the case of weight quantization, the advantage would be to save space in terms of model size. However , if we saved the model state with torch.save(quant_weight_lenet.state_dict(),'qw_lenet.pt') we would notice that it consumes the same amount of memory as its floating-point variant. That is because Brevitas is not concerned with deploying quantized models efficiently on its own. In order to deploy the model efficiently, we have to export it to an inference framework/toolchain first."
But apart from torch.save(), how should I export my training results to an inference framework/toolchain.

**Comment by YZW-explorer** [comment]
Sincere thanks

**Comment by volcacius** [comment]
The rest of the README shows a couple of example on how to export to either [onnx](https://github.com/Xilinx/brevitas/#export-to-standard-onnx) or [pytorch itself](https://github.com/Xilinx/brevitas/#export-to-pytorch-quantized-inference-ops). Just keep in mind in general it's still experimental and only some layers and precision are supported.
